bridgesync/processor.go

```diff
@@ -506,8 +508,9 @@ func (p *processor) GetBridgesPaged(
 	bridgeResponsePtrs := make([]*BridgeResponse, len(bridgePtrs))
 	for i, bridgePtr := range bridgePtrs {
 		bridgeResponsePtrs[i] = &BridgeResponse{
-			Bridge:     *bridgePtr,
-			BridgeHash: bridgePtr.Hash(),
+			Bridge:      *bridgePtr,
+			BridgeHash:  bridgePtr.Hash(),
+			NativeToken: bridgePtr.GasTokenAddress == bridgePtr.OriginAddress,
 		}
```

Comment:
We should check if the `OriginAddress` is `0x0`, in that case it is also a native token bridge transfer.
As a side note, consider extracting the `common.HexToAddress("0x0")` to some global var.

```suggestion
		isNativeToken := bridgePtr.OriginAddress == bridgePtr.GasTokenAddress ||
			bridgePtr.OriginAddress == common.HexToAddress("0x0")

		bridgeResponsePtrs[i] = &BridgeResponse{
			Bridge:      *bridgePtr,
			BridgeHash:  bridgePtr.Hash(),
			NativeToken: isNativeToken,
		}
```